src/Halcyon/Datasource/FileDatasource.php

```diff
@@ -31,6 +31,13 @@ class FileDatasource extends Datasource implements DatasourceInterface
      */
     protected $files;
 
+    /**
+     * Resolved path map.
+     *
+     * @var array
+     */
+    protected $pathResolverMap = [];
```

Comment:
@jaxwilko Could we change this to `$resolvedBasePaths`? More clearly indicates what's being stored here. You'll need to change the references further on in the class too.

```suggestion
     * A list of resolved base paths.
     *
     * @var array
     */
    protected $resolvedBasePaths = [];
```

Reply:
Happy with this change too, I've got a bunch of debug code in there atm for another PR I'm looking at but I'll try and get that one sorted and make this change later today :)

Reply:
This has been changed in https://github.com/wintercms/storm/pull/34/commits/faece692a56b3544d4d0ee4318b865dd2eef7606